src/grid.hoc.js

```diff
@@ -1,85 +0,0 @@
-// @flow
```

Comment:
I'm in favor of removing `layout.hoc`. For `grid.hoc` I can see it both ways so this is fine. My rationale is that GridHOC may be used in styleguide components. Ultimately though it may be easier to think of this library as just 2 components and then wrap anything on your styleguide with one of them